CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Breaking changes
```

Comment:
Keep A Changelog currently doesn't have a separate section for breaking changes (see also discussion [here](https://github.com/olivierlacan/keep-a-changelog/issues/41)). For now the format we're using is that the change should be under `Changed` and have `BREAKING:` as a prefix (see e.g. [here](https://github.com/edgehog-device-manager/edgehog/blob/main/CHANGELOG.md#changed))